File: 5주차/0706/찬영/숫자 문자열과 영단어.java
public int solution(String s){

    String[] strArray = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight",
    "nine"};
    String[] numArray = {"0", "1", "2", "3", "4", "5", "6", "7", "8", "9"};

    for (int i = 0; i < strArray.length; i++) {
      s = s.replaceAll(strArray[i], numArray[i]);
    }
    return Integer.parseInt(s);

File: 5주차/0706/찬영/숫자 문자열과 영단어.py
def solution(s):
    numbers = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for index, num in enumerate(numbers):
        if num in s:
            s=s.replace(num,str(index))
    return int(s)